contrib/auto-completion/snippets/emacs-lisp-mode/.yas-parents

```diff
@@ -0,0 +1 @@
+prog-mode
```

Comment:
Looked it up, it makes emacs lisp also able to access prog-mode snippets:
https://capitaomorte.github.io/yasnippet/snippet-organization.html#sec-3